tfhe/src/core_crypto/entities/glwe_ciphertext_list.rs

```diff
@@ -67,6 +67,26 @@ impl<Scalar: UnsignedInteger, C: Container<Element = Scalar>> GlweCiphertextList
     /// assert_eq!(glwe_list.glwe_ciphertext_count(), ciphertext_count);
     /// assert_eq!(glwe_list.ciphertext_modulus(), ciphertext_modulus);
     ///
+    /// // Alternatively, create a new GlweCiphertextList and fill it using a function
+    /// let glwe_list = GlweCiphertextList::from_fn(
+    ///     glwe_size,
+    ///     polynomial_size,
+    ///     ciphertext_count,
+    ///     ciphertext_modulus,
+    ///     |i, j| {
+    ///         // The `i` value represents the index in the list being filled;
+    ///         // The `j` value represents the index in the ciphertext being filled;
+    ///         // In this example, for every index pair `(i, j)`, we fill the
+    ///         // corresponding value using the formula `i + j`
+    ///         (i + j as u64)
```

Comment:
likely fix is (i + j) as u64

Reply:
I forgot about the precedence, it should be working now.